src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
+* Operator and individual applications have separate service accounts with appropriate privileges
+* For Openshift, Lighthouse CoreDNS metrics are integrated with other CoreDNS metrics
+* KIND deployments support OVS CNI
+* `submariner_connections` is renamed to `submariner_requested_connections` in Prometheus for OpenShift clusters
+* Clusters can be connected with only Lighthouse and no dataplane connectivity via subctl
+* Added support for Flannel in k3s deployments
+* All Submariner metrics are prefixed with `submariner_` in Prometheus for OpenShift clusters
+* Services with Global IPs can now be exported
```

Comment:
```suggestion
* Services with Global IPs can now be exported
```

Hmm @vthapar @sridhargaddam  ^  

I think it's headless services, services where possible already

Reply:
The sentence should be framed to convey the following message.

When using Globalnet, only exported services will be annotated with globalIP and will be made accessible to remote clusters.

Reply:
```suggestion
* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated every Service in the cluster, whether or not it was exported.
```